lib/babushka/system_definition.rb

```diff
@@ -72,6 +72,8 @@ def codename_str
           '15.04' => :vivid,
           '15.10' => :wily,
           '16.04' => :xenial,
+          '17.10' => :artful,
```

Comment:
Shouldn't we also add 16.10 (Yakkety Yak) and 17.04 (Zesty Zapus) while we are at it?

Reply:
For completeness, I suppose, although those releases are at this point end of life. But to me the best solution would be not needing to maintain a list like this. /cc @benhoskings

Reply:
End of life doesn't mean, they aren't still used...

But yeah, this list does not need to be here, does it? Because `DISTRIB_CODENAME` within `/etc/lsb-release` file should work. Although for some ubuntu derivates like elementaryOS who have their own codename, but need the ubuntu xenial serves for packages, we still might need a solution.

Reply:
I agree, this list shouldn't be here - much better to accept any version from the system.